mor/mor/feel/feelmor/toolboxmor_impl.hpp

```diff
@@ -246,7 +246,8 @@ ToolboxMor<SpaceType, Options>::setupSpecificityModel( boost::property_tree::ptr
         Feel::cerr << "Warning!! the database does not contain the property file! Expect bugs!"
                    << std::endl;
 
-    M_modelProperties = std::make_shared<ModelProperties>(M_propertyPath);
+    M_modelProperties = std::make_shared<ModelProperties>(); // TODO : put directoryLibExpr and worldcomm
```

Comment:
it was just a comment in order to don't forget that on online mode, the expression will be generated (because not saved in db). Maybe we can also add symbolic expr lib inside and so directoryLibExpr.